core/src/main/java/com/linecorp/armeria/server/DefaultServiceRequestContext.java

```diff
@@ -534,66 +536,50 @@ private void updateAdditionalResponseHeaders(
         }
     }
 
-    @Override
-    public boolean removeAdditionalResponseHeader(CharSequence name) {
-        return removeAdditionalResponseHeader(additionalResponseHeadersUpdater, name);
-    }
```

Comment:
I don't get why this method has to be removed. Doesn't a user sometimes need to remove a header from the additional response headers? e.g. A user can write a decorator that removes an additional response header added by other service.

Reply:
I thought the situation is never going to happen but I guess I was wrong. 😆
And I really like the idea of providing the API to mutate headers. So we can have this behavior again. Thanks!